src/platform/libretro/libretro.c

```diff
@@ -1480,7 +1480,7 @@ static void _setupMaps(struct mCore* core) {
 		/* Map VRAM */
 		descs[i].ptr    = gb->video.vram;
 		descs[i].start  = GB_BASE_VRAM;
-		descs[i].len    = GB_SIZE_VRAM;
+		descs[i].len    = GB_SIZE_VRAM_BANK0;
```

Comment:
This probably depends on if it's GB (`GB_SIZE_VRAM_BANK0`) or GBC (`GB_SIZE_VRAM`). I don't believe I have an internal way of checking this, so you may just have to do a model check.

Reply:
we can only mapped the 1st bank here anyways, since for one cheevos memmap does not work with bank switching (hence cgb's wram is split). mapping the entire 32K cgb vram would overlap with SRAM address.